syntrax/build.gradle

```diff
@@ -0,0 +1,12 @@
+repositories {
+    mavenLocal()
+    mavenCentral()
+    jcenter()
+}
+
+dependencies {
+    implementation project(':server')
+    testImplementation project(':server')
+    testImplementation project(path: ':server', configuration: 'tests')
+    implementation 'org.atp-fivt:jsyntrax:1.31'
```

Comment:
1.32 is deployed to public Maven repository with a public constructor for `InputArguments`. Please change to 1.32

Reply:
done